### ISSUE TITLE
Need flag to disable setting system PATH variable

### ISSUE THREAD

**Issue by alexchandel** [StrawberryPerl/Perl-Dist-Strawberry#211]
The installer currently adds the Strawberry Perl bin folders to the system PATH environment variable.  This is normally ok, but I need the ability to customize this.  What is the installer flag to disable setting the PATH?

**Comment by shawnlaffan** [comment]
I'm not sure exactly how to do this, but the WIX installer (version 3) is used to build the MSI files.  The relevant files are in https://github.com/StrawberryPerl/Perl-Dist-Strawberry/tree/master/share/msi

You could alternately use the portable versions as these do not add to the global path.  

Berrybrew is also a good option.  It uses portable versions and includes an option to turn off the current perl.  https://github.com/dnmfarrell/berrybrew

And as a general note, removing the Strawberry Perl path components typically needs to be an all or nothing approach.  Many of the libraries installed with Strawberry Perl will not run unless they can access the DLLs in the `...\c\bin` directory.  (There are ways around this but nothing by default).

**Comment by alexchandel** [comment]
Agreed that it is all or nothing.  The envisioned case here is one where the caller would have the PATH set up to the requirements when calling Strawberry Perl.

[This line](https://github.com/StrawberryPerl/Perl-Dist-Strawberry/blob/8ec82760ac883c7a6a00425fc2ec53967526b528/share/msi/MSI_main-v2.wxs.tt#L171) seems to be where the PATH is appended.  Is there a "silent"/"unattended" install flag?

**Comment by shawnlaffan** [comment]
> Is there a "silent"/"unattended" install flag?

Maybe this?  https://learn.microsoft.com/en-us/windows/win32/msi/standard-installer-command-line-options#quiet

**Comment by alexchandel** [comment]
Yes you are right, the standard `msiexec` flags work.  (They must be supplied in the right order, `msiexec /package C:\path\to\thing.msi /passive`.)  `PERL_PATH=Yes` could be appended after that for ActiveState MSIs, but still trying to figure out where these flags get created in Wix ...

**Comment by shawnlaffan** [comment]
The answer is probably the same as in https://github.com/StrawberryPerl/Perl-Dist-Strawberry/issues/224#issuecomment-2450696219

https://github.com/StrawberryPerl/Perl-Dist-Strawberry/tree/master/share/msi

I have not worked through these files or the MSI process in any detail, so any findings you can note would be very helpful.
